wiring/src/spark_wiring_ble.cpp

```diff
@@ -2192,6 +2186,7 @@ ssize_t BleLocalDevice::getScanResponseData(BleAdvertisingData* scanResponse) co
         return SYSTEM_ERROR_INVALID_ARGUMENT;
     }
     scanResponse->clear();
+    scanResponse->resize(BLE_MAX_ADV_DATA_LEN);
```

Comment:
Can potentially fail, `hal_ble_gap_get_scan_response_data` may write out of bounds.

Reply:
Added check for resize failure.